migrations/versions/0420_bilingual_templates.py

```diff
@@ -0,0 +1,45 @@
+"""
+
+Revision ID: 0420_bilingual_templates
+Revises: 0419_take_part_in_research
+Create Date: 2023-08-30 17:26:28.925639
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+revision = "0420_bilingual_templates"
+down_revision = "0419_take_part_in_research"
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("templates", sa.Column("welsh_content", sa.Text(), nullable=True))
+    op.add_column("templates", sa.Column("welsh_subject", sa.Text(), nullable=True))
+    template_language_options = postgresql.ENUM(
+        "english", "welsh_then_english", name="template_language_options", create_type=False
+    )
+    template_language_options.create(op.get_bind(), checkfirst=True)
```

Comment:
Ah nice, I wasn't super aware of the `create_type=False` and `checkfirst=True`. I assume that makes local testing a bit easier if you have to re-run the migrations a few times. We wouldn't expect to ever need those options in prod, right? (but they're fine to stay IMO)

Reply:
You used `create_type=False` in migration 0413, which was part of my inspiration 😁 . Other part was stackoverflow, as the auto-created migration file didn't work. Check my commit for this code for details ☺️

Reply:
Oh 😂  haha! Well that is proof that my memory is shot then ... 😬